data-service/authentication/docker.sh

```diff
@@ -0,0 +1 @@
+docker run -p 9990:8080 -e KEYCLOAK_USER=admin -e KEYCLOAK_PASSWORD=admin -e KEYCLOAK_IMPORT=/tmp/unified-codes.json --mount type=bind,source="$(pwd)",target=/tmp jboss/keycloak
```

Comment:
For maintainability and testing etc. thinking that instead of importing, it might be helpful to have some scripts which generate the realm config? Would also considerably reduce the number of lines being committed!

![please](https://user-images.githubusercontent.com/43223637/88993712-ca129200-d33a-11ea-8cba-70a3e1da0552.gif)

Reply:
54ab5ab
mac only at the moment - running docker in a sep terminal so that I can run it all at once.
Also the `Realm Role` assignment doesn't work. Other users report that it worked in previous versions 🤷‍♂️ 
For now - manual role assignment. The alternative is to request user details and extract the user guid so that roles can be assigned. not sure that's warranted.